<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonFloorProperties">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>450</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Floor Properties</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTreeView" name="treeView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::ActionsContextMenu</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QVTKOpenGLNativeWidget" name="vtkWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_File"/>
    <addaction name="actionNew_Group"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionItem_Options"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_File"/>
   <addaction name="actionItem_Options"/>
   <addaction name="actionNew_Group"/>
   <addaction name="actionDelete_File"/>
   <addaction name="actionSearch_Items"/>
  </widget>
  <action name="actionOpen_File">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Downloads/open.png</normaloff>:/Downloads/open.png</iconset>
   </property>
   <property name="text">
    <string>Open File</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionItem_Options">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Downloads/options.png</normaloff>:/Downloads/options.png</iconset>
   </property>
   <property name="text">
    <string>Item Options</string>
   </property>
   <property name="toolTip">
    <string>Item Options</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionNew_Group">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Downloads/add.png</normaloff>:/Downloads/add.png</iconset>
   </property>
   <property name="text">
    <string>New Group</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionDelete_File">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Downloads/delete.png</normaloff>:/Downloads/delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete File</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionSearch_Items">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Downloads/search.png</normaloff>:/Downloads/search.png</iconset>
   </property>
   <property name="text">
    <string>Search Items</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLNativeWidget</class>
   <extends>QWidget</extends>
   <header>qvtkopenglnativewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
